3scale/porta
# Configuration of min_infix_len to address THREESCALE-5123 - Improve Admin Portal Search feature

**What this PR does / why we need it**
Improve the Admin Portal User search feature in order to find users by email address even in case the address contains two-character strings separated by dots (e.g. test@mail.co.uk)
This isn't currently possible due to `min_infix_len` being set to `3` in `thinking_sphinx.yml`

**Which issue(s) this PR fixes** 
[THREESCALE-5123](https://issues.redhat.com/browse/THREESCALE-5123) (RFE), THREESCALE-632 (bug)

**Verification steps** 
1. Go to the Admin Portal and create a user with email `test@aaa.bb.com`
2. Force a reindex (or wait for it to happen automatically)
3. Make sure the user can be found via the seach feature by inserting `test@aaa.bb.com` in the search bar
